cubedash/summary/_stores.py

```diff
@@ -815,6 +777,95 @@ def search_items(
         if require_geometry:
             query = query.where(DATASET_SPATIAL.c.footprint != None)
 
+        return query
+
+    def get_count(
+        self,
+        product_names: Optional[List[str]] = None,
+        time: Optional[Tuple[datetime, datetime]] = None,
+        bbox: Tuple[float, float, float, float] = None,
+        dataset_ids: Sequence[UUID] = None,
+        require_geometry=True,
+    ) -> int:
+        """
+        Do the most simple select query to get the count of matching datasets.
+        """
+        query: Select = select([func.count()]).select_from(DATASET_SPATIAL)
+
+        query = self._add_fields_to_query(
+            query,
+            product_names=product_names,
+            time=time,
+            bbox=bbox,
+            dataset_ids=dataset_ids,
+            require_geometry=require_geometry,
+        )
+
+        result = self._engine.execute(query).fetchall()
+
+        if len(result) != 0:
+            return result[0][0]
+        else:
+            return 0
```

Comment:
It's only cosmetic, but sqlalchemy has a neat `engine.execute(query).scalar()` function to query a single number succinctly.

Reply:
Aha, thanks.